packages/ui-keyring/src/ledger/index.ts

```diff
@@ -46,43 +46,9 @@ export default class Ledger {
     this.#transport = transport;
   }
 
-  private async _getApp (): Promise<SubstrateApp> {
-    if (!this.#app) {
-      const def = transports.find(({ type }) => type === this.#transport);
-
-      assert(def, `Unable to find a transport for ${this.#transport}`);
-
-      const transport = await def.create();
-
-      this.#app = APPS[this.#chain](transport);
-    }
-
-    return this.#app;
-  }
-
-  private async _withApp <T> (fn: (app: SubstrateApp) => Promise<T>): Promise<T> {
-    try {
-      const app = await this._getApp();
-
-      return await fn(app);
-    } catch (error) {
-      this.#app = null;
-
-      throw error;
-    }
-  }
-
-  private async _wrapError <T extends ResponseBase> (promise: Promise<T>): Promise<T> {
-    const result = await promise;
-
-    assert(result.return_code === SUCCESS_CODE, result.error_message);
-
-    return result;
-  }
-
   public async getAddress (confirm = false, accountOffset = 0, addressOffset = 0, { account = LEDGER_DEFAULT_ACCOUNT, addressIndex = LEDGER_DEFAULT_INDEX, change = LEDGER_DEFAULT_CHANGE }: Partial<AccountOptions> = {}): Promise<LedgerAddress> {
-    return this._withApp(async (app: SubstrateApp): Promise<LedgerAddress> => {
-      const { address, pubKey } = await this._wrapError(app.getAddress(account + accountOffset, change, addressIndex + addressOffset, confirm));
+    return this.#withApp(async (app: SubstrateApp): Promise<LedgerAddress> => {
```

Comment:
Slipped-in, was a small cosmetic change that I inadvertently committed, but all good, doesn't need to go later.